trikot-viewmodels-declarative/compose/src/main/kotlin/com/mirego/trikot/viewmodels/declarative/compose/viewmodel/VMDList.kt

```diff
@@ -21,6 +21,7 @@ import com.mirego.trikot.viewmodels.declarative.compose.extensions.isOverridingA
 import com.mirego.trikot.viewmodels.declarative.compose.extensions.observeAsState
 import com.mirego.trikot.viewmodels.declarative.content.VMDIdentifiableContent
 
+@Deprecated("Use either VMDLazyColumn or VMDLazyRow instead")
```

Comment:
I would say **Use VMDLazyColumn** as this was backed by a LazyColumn and did not support horizontal anyways.